### ISSUE TITLE
"Download of bootstrap.block has finished".

### ISSUE THREAD

**Issue by vi** [Tribler/tribler#4460]
##### Tribler version/branch+revision: #####

04d5a2880235f6ba03023e63cf06baa25fafdc8f  release-7.3.0-beta2

##### Operating system and version: #####

Linux

##### Steps to reproduce the behavior: #####

1. Use Tribler 7.3.0-beta1
2. Update to beta2
3. Start
4. Wait

"Enable developer mode" is on in settings. "Enable token mining" is also on.

##### Actual behavior: #####

"Download finished" "Download of bootstrap.block has finished" baloon message appears. There is no such download visible in the list.

##### Expected behavior: #####

Nothing suspicious. Or special internal bookkeeping downloads are visible when "Enable developer mode" is enabled (maybe in a special debugging tab after "... [COMPLETED] [ACTIVE] [INACTIVE]". But not a baloon notification without actual inspectable download.

**Comment by devos50** [comment]
We should indeed hide this download. @grimadas will most likely fix this somewhere next week 👍

**Comment by vi** [comment]
If Tribler routinely uses internal hidden downloads then there should be a way to show them for debugging.

**Comment by devos50** [comment]
Yes, it might be a good idea to show something like a `hidden` tab on the downloads overview when developer mode is enabled. But that also means that we should return these downloads in our request to `GET /downloads`. Maybe we can add a flag to this endpoint to also return hidden downloads?

**Comment by vi** [comment]
Or just return all downloads (uless number of hidden downloads is expected to be order of magnitute more than usual downloads) with a `"hidden":true` metadata and GUI filters it (or puts into separate tab).

**Comment by vi** [comment]
Does anonymity settings affect the way that "bootstrap.block" is downloaded?

Sometimes BitTorrent protocol is just blocked and enabling 1 hop is a way to workaround.

**Comment by devos50** [comment]
Fixed by #4466

**Comment by devos50** [comment]
I observed this issue again.
